beacon_node/http_api/tests/fork_tests.rs

```diff
@@ -293,7 +293,7 @@ async fn sync_committee_indices_across_fork() {
         400
     );
 
-    // Once the head is updated it should be useable for requests, including in the next sync
+    // Once the head is updated it should be usable for requests, including in the next sync
```

Comment:
```suggestion
    // Once the head is updated it should be useable for requests, including in the next sync
```

Useable is fine in en_GB and en_AU